### DISCUSSION TITLE
[Projects beta]: Progress of tasks visible in a card

### DISCUSSION THREAD

**Discussion by amrap030** [community/community#7240] [Projects and Issues]
In the old project board, it was possible to see the progress of a list of tasks, if the issue contained a list of tasks. In the new Projects Beta board this feature is missing.

**Comment by ErisDS** [comment]
This is something we're missing as well.

**Reply by ddelange** [reply]
duplicate of https://github.com/github/feedback/discussions/7575 ?

**Comment by Jorsta** [comment]
Agree! Please make task lists visible I board cards. At least so that progress is shown as before. Or – even better – with checkboxes visible and check-uncheckable in the board view (as with notes in non-beta boards).

**Comment by micahalcorn** [comment]
Please also consider adding this to the table view. It would be great to be able to add a column that would depict something along these lines:

<img width="139" alt="Screen Shot 2021-12-07 at 10 51 51 PM" src="https://user-images.githubusercontent.com/273937/145162293-880fb105-10a3-40aa-a60b-2a074c0d3eee.png">
